Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiSwitcher.h

```diff
@@ -40,6 +40,9 @@ namespace AzToolsFramework::ViewportUi::Internal
         void SetButtonTooltip(ButtonId buttonId, const AZStd::string& tooltip);
 
     private:
+        //! Removes color from switcher icon.
+        QPixmap RemoveIconColor(const char* buttonIcon);
```

Comment:
suggestion: It doesn't look like this needs to be a member function so we could just move this to the .cpp as a static function (or leave a free function interface in the .h so we could test it separately 🙂)